common/reviews/api/api-extractor-model.api.md

```diff
@@ -613,12 +613,18 @@ export class Excerpt {
 
 // @public (undocumented)
 export class ExcerptToken {
-    constructor(kind: ExcerptTokenKind, text: string);
+    constructor(kind: ExcerptTokenKind, text: string, canonicalReference?: DeclarationReference);
+    // (undocumented)
+    readonly canonicalReference: DeclarationReference | undefined;
+    // (undocumented)
+    static fromJSON(object: IExcerptToken): ExcerptToken;
```

Comment:
Similar to [.NET](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions), we use `camelCase` for acronyms with more than 2 letters.  So the name would be `fromJson` instead of `fromJSON`.

Reply:
However, ECMAScript specifically defines `toJSON` with upper-case for `JSON.parse`/`JSON.stringify`. Having a `toJSON` and a `fromJson` seems confusing. I added them for convenience, but I could always inline them into their only usages in `ApiDeclaredItem`.

Reply:
I've just removed the members and inlined them into their use sites.

Reply:
Cool 👍